src/main/java/Duke/AddDeadlineCommand.java

```diff
@@ -0,0 +1,41 @@
+package Duke;
```

Comment:
Perhaps you could group the class related to ICommand under one package like duke.command? It might help the reader to find the respective class?